src/plugins/intel_cpu/thirdparty/FindACL.cmake

```diff
@@ -0,0 +1,260 @@
+# Copyright (C) 2018-2022 Intel Corporation
```

Comment:
looks like we need to revert update of open_model_zoo

Reply:
Thanks, reverted